env.sh

```diff
@@ -15,3 +15,5 @@ export ZLIB_HASH=ff0ba4c292013dbc27530b3a81e1f9a813cd39de01ca5e0f8bf355702efa593
 export LIBEVENT_HASH=92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
 export OPENSSL_HASH=cf3098950cb4d853ad95c0841f1f9c6d3dc102dccfcacd521d93925208b76ac8
 export TOR_HASH=59bb7d8890f6131b4ce5344f3dcea5deb2182b7f4f10ff0cb4e4d81f11b2cf65
+
+export DOCKER="$(if command -v podman >/dev/null 2>&1; then echo podman; else echo docker; fi)"
```

Comment:
Good point.

Maybe we could change the `env.sh` blurb so that it still defaults to `docker`, but then if `$DOCKER` is already defined in the environment, it doesn't overwrite it?

That way, if you want to build with `podman`, you'd simply do:
```
$ export DOCKER=podman
$ source ./env.sh
$ ./build_linux.sh
```